src/index.html

```diff
@@ -6,9 +6,83 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"rel="stylesheet"
+      >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-img">
+        <img src="images/logo.png" alt="Moyo">
+      </a>
+      <nav class="nav>">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a class="nav-link is-active"
+            href="#"
+            >
+            Apple
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link"
+            href="#"
+            >
```

Comment:
```suggestion
            <a 
              class="nav-link"
              href="#"
            >
```
Consider fixing indents according to the code style guide. Also, try to fix it further in the code